src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas1/Cas1SpaceBookingDaySummaryTransformer.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas1
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBookingCharacteristic
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBookingDaySummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.PersonSummary
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas1SpaceBookingDaySummarySearchResult
+
+@Component
+class Cas1SpaceBookingDaySummaryTransformer {
+
+  fun toCas1SpaceBookingDaySummary(
+    jpa: Cas1SpaceBookingDaySummarySearchResult,
+    personSummary: PersonSummary,
+  ) = Cas1SpaceBookingDaySummary(
+    id = jpa.id,
+    canonicalArrivalDate = jpa.canonicalArrivalDate,
+    canonicalDepartureDate = jpa.canonicalDepartureDate,
+    tier = jpa.tier,
+    releaseType = jpa.releaseType,
+    essentialCharacteristics = characteristicEntityToCharacteristic(jpa.characteristics),
```

Comment:
can we please rename the `Cas1SpaceBookingDaySummarySearchResult.characteristics` to `Cas1SpaceBookingDaySummarySearchResult.characteristicsPropertyNames` for clarity?